rs/canonical_state/src/encoding/tests/compatibility.rs

```diff
@@ -573,6 +573,8 @@ fn canonical_encoding_request_v18_plus() {
 ///       05                      # field_index(Request::method_payload)
 ///       41                      # bytes(1)
 ///          06                   # "\x06"
+///       07                        # unsigned(7)
+///       A2                        # map(2)
```

Comment:
I believe this should be (but you should double-check):

```suggestion
///       07                      # field_index(Request::metadata)
///       A0                      # map(0)
```

Reply:
Done.